packages/lib-classifier/src/store/utils/session.js

```diff
@@ -1,19 +1,12 @@
 import hash from 'hash.js'
 
-const storage = window.sessionStorage || window.localStorage
+const storage = window.sessionStorage
 
 const sessionUtils = {
-  fiveMinutesFromNow () {
-    const d = new Date()
-    d.setMinutes(d.getMinutes() + 5)
-    return d
-  },
-
   generateSessionID () {
     const sha2 = hash.sha256()
     const id = sha2.update(`${Math.random() * 10000}${Date.now()}${Math.random() * 1000}`).digest('hex')
-    const ttl = this.fiveMinutesFromNow()
-    const stored = { id, ttl }
+    const stored = { id }
     try {
       storage.setItem('session_id', JSON.stringify(stored))
```

Comment:
[rambling][minor] Curiously, this means that we're storing session ID in an unnecessary "object layer", i.e. `session_id = "{ id: \"1234\" }"` instead of `session_id = "1234"`. Keeping it this way is fine by me, though. (And even if we did change it to the latter format, I wonder what'd happen if a user with an existing session ID refreshes their page after we push the changes. 🤔 )

Reply:
Remember that this stored object is also going to be read by PFE, so anything written to storage here must be readable by PFE.

Reply:
If PFE does generate a new session when you go to Talk, that would be a breaking change, so it's worth testing that case.